src/test/datascience/data-viewing/dataViewerDependencyServic.node.unit.test.ts

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is the same file that was in `src/test/datascience/data-viewing/dataViewerDependencyServic.node.test.ts`. Only that `checkAndInstallMissingDependencies` now receives an object.